osbuild/mounts.py

```diff
@@ -126,8 +126,20 @@ def __init__(self, args):
     # pylint: disable=no-self-use
     @abc.abstractmethod
     def translate_options(self, options: Dict) -> List:
-        if options.get("readonly", False):
-            return ["-o", "ro"]
+        opts = []
+        for key, value in options.items():
+            if key == "readonly":
+                if value:
+                    opts += ["ro"]
+            elif key == "defaults":
+                if value:
+                    opts += ["defaults"]
+            elif key in ["uid", "gid", "umask", "shortname"]:
+                opts += [f"{key}={value}"]
+            else:
+                raise RuntimeError(f"Unsupported mount option {key}")
```

Comment:
This shouldn't be needed.  The schema validates the options so osbuild will never reach this stage if any other option is specified.

Reply:
On the pathway from `image-info` there's no schema validation. Should I implement a schema validation when I'm using a mounter in `image-info` instead of Raising an exception here ?

Reply:
I think that would be better in image-info, yeah.

Reply:
It's done.